crates/fuzzing/src/oracles/v8.rs

```diff
@@ -0,0 +1,289 @@
+use super::{first_exported_function, first_exported_memory, log_wasm};
+use rusty_v8 as v8;
+use std::convert::TryFrom;
+use std::sync::Once;
+use wasmtime::*;
+
+/// Performs differential execution between Wasmtime and V8.
+///
+/// This will instantiate the `wasm` provided, which should have no host
+/// imports, and then run it in Wasmtime with the `config` specified and V8 with
+/// default settings. The first export is executed and if memory is exported
+/// it's compared as well.
+pub fn differential_v8_execution(wasm: &[u8], config: &crate::generators::Config) -> Option<()> {
```

Comment:
Can you add a comment here that it is the caller's responsibility to ensure that the given wasm won't infinitely loop, since we have no way of adding fuel/timeouts to v8, afaik? Alternatively, in the past I've tried to encode that in the type system by making the oracle a tiny bit less reusable and taking a `wasm_smith::Module` (or `wasm-opt -ttf` module in the before times) rather than a raw byte slice.